resources/project_group.md

```diff
@@ -49,7 +49,12 @@ status 200 OK
 {
     "meta": {
         "status": 200,
-        "record_count": 2
+        "record_count": 65,
+        "page_count": 3,
+        "next_page": "http://platform.api.onesky.io/1/project-groups?per_page=25&page=3", *
```

Comment:
The `*` here reference to the remark below
I think it's fine to have a remark section like [file list](https://github.com/onesky/api-documentation-platform/blob/master/resources/file.md#list---list-uploaded-files) and remove the `*` here